src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,14 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String model;
+
+    public Bulldozer(String model) {
+        this.model = model;
+    }
+
     public Bulldozer() {
```

Comment:
The no-args constructor is good to have but make sure you don't add unnecessary empty lines within method or constructor bodies.